.env

```diff
@@ -1,4 +1,5 @@
 REACT_APP_STUDENT_SVC_BASE_URL=http://api.judgegirl.beta.pdlab.csie.ntu.edu.tw
 REACT_APP_EXAM_SVC_BASE_URL=http://api.judgegirl.beta.pdlab.csie.ntu.edu.tw
 REACT_APP_PROBLEM_SVC_BASE_URL=http://api.judgegirl.beta.pdlab.csie.ntu.edu.tw
-REACT_APP_SUBMISSION_SVC_BASE_URL=http://api.judgegirl.beta.pdlab.csie.ntu.edu.tw
\ No newline at end of file
+REACT_APP_SUBMISSION_SVC_BASE_URL=http://api.judgegirl.beta.pdlab.csie.ntu.edu.tw
+REACT_APP_HOMEWORK_SVC_BASE_URL=http://api.judgegirl.beta.pdlab.csie.ntu.edu.tw
```

Comment:
Reduce `REACT_APP_EXAM_SVC_BASE_URL` and `REACT_APP_HOMEWORK_SVC_BASE_URL` into 
`REACT_APP_ACADEMY_SVC_BASE_URL=http://api.judgegirl.beta.pdlab.csie.ntu.edu.tw`

remember to also change the services which depend on the exam or homework service's link

Reply:
should also modify the exam service